README.md

```diff
@@ -34,7 +34,7 @@ For an existing react application, follow the steps below:
 1. Load the fiori-fundamentals styles. If using create-react-app, this will be in `App.css`.
 
     ```scss
-    @import '~fiori-fundamentals/dist/fiori-fundamentals.min.css';
+    @import '~fiori-fundamentals/dist/fiori-fundamentals-ie11.min.css';
```

Comment:
This is probably a Fiori Fundamentals question, but why does the 1.4.3 distribution package have both the `-ie11` CSS file and the non-`-ie11` CSS file?  If it needs to support IE 11, why isn't _that_ just the only CSS file distributed?

Reply:
I wondered the same thing, see https://github.com/SAP/fundamental/issues/1244#issuecomment-459779074